csrc/serde/fusion_cache.fbs

```diff
@@ -345,6 +345,7 @@ table CudaKernel {
 
 // Each Fusion Executor maps to a lowered and compiled kernel.
 table FusionExecutor {
+  is_compilation_skipped : bool;
```

Comment:
Why is this needed? isCompilationSkipped is a method that describes the state of the FusionExecutor. Making it a field in `table FusionExecutor` seems to suggest it's part of the state itself. cc @rdspring1

Reply:
Sorry for the delay -- I missed the notification. 

I understood how it's used [here](https://github.com/NVIDIA/Fuser/pull/1930/files#diff-e72d395ea8a3ab3d2a909dfb3677beac1bbd89336b54ee858867f652b48be4f0R2339) to avoid unnecessary compilation. However, can we replace that check with a check on the fusion itself instead of introducing yet another flag? If works, this can avoid an additional state that has to be consistent with some existing state of the fusion.